stock_reserve/model/stock_reserve.py

```diff
@@ -108,8 +108,13 @@ def get_location_from_ref(self, ref):
 
     @api.model
     def _default_picking_type_id(self):
-        ref = "stock.picking_type_out"
-        return self.env.ref(ref, raise_if_not_found=False).id
+        domain = [
```

Comment:
@acsonefho I would have changed to the company default warehouse 'out_type_id' field.

Reply:
There is an implicit order on sequence, id in the search for stock.picking.type, so you can control which one gets selected by default. I'm fine with this. 

The alternative is to rely on the warehouse sequence for a given company, this is also fine. 

Whatever the choice, I would document it in the README.